arwen/elrondapi/elrondei.go

```diff
@@ -69,7 +72,23 @@ func ElrondEIImports() (*wasmer.Imports, error) {
 	imports := wasmer.NewImports()
 	imports = imports.Namespace("env")
 
-	imports, err := imports.Append("getOwner", getOwner, C.getOwner)
+	// TODO: rename getOwner -> getSCAddress when all affected contracts (delegation etc.) are ready
+	imports, err := imports.Append("getOwner", getSCAddress, C.getSCAddress)
```

Comment:
change it rigth now, contracts will follow this PR :)

Reply:
Fixed.